test/ReCaptchaTest.php

```diff
@@ -163,4 +157,47 @@ public function testUsesReCaptchaHelper()
         $captcha = new ReCaptcha;
         $this->assertEquals('captcha/recaptcha', $captcha->getHelperName());
     }
+
+    public function testValidationForDifferentElementName()
+    {
+        $captcha = new ReCaptcha([
+            'site_key' => getenv('TESTS_ZEND_SERVICE_RECAPTCHA_SITE_KEY'),
+            'secret_key' => getenv('TESTS_ZEND_SERVICE_RECAPTCHA_SECRET_KEY'),
+        ]);
+        $captcha->getService()->setIp('127.0.0.1');
+        $captcha->getService()->setHttpClient($this->getHttpClient());
+
+        $response = getenv('TESTS_ZEND_SERVICE_RECAPTCHA_RESPONSE');
+        $value = 'g-recaptcha-response';
+        $context = ['g-recaptcha-response' => getenv('TESTS_ZEND_SERVICE_RECAPTCHA_RESPONSE')];
+
+        $this->assertTrue($captcha->isValid($value, $context));
+    }
+
+    public function testValidationForResponseElementName()
+    {
+        $captcha = new ReCaptcha([
+            'site_key' => getenv('TESTS_ZEND_SERVICE_RECAPTCHA_SITE_KEY'),
```

Comment:
Should probably skip these tests if either of these variables are not defined (see comment above about committed secrets)